### DISCUSSION TITLE
Is client.user.verified bugged?

### DISCUSSION THREAD

**Discussion by JavascriptLearner815** [discordjs/discord.js#5226] [Q&A] [answered]
Here's the narrowed-down code:

```javascript
let verified = 'Nah...';

if (client.user.verified) verified = 'Yep!!!';
```

It returns 'Yep!!!' even though I definitely know that my bot ISN'T verified.

![Discord.js Bug](https://user-images.githubusercontent.com/59782978/104958723-23788700-5996-11eb-9d3a-2a20cd1f48ee.PNG)

Is this verified being mistaken for unverified, or something wrong on my end?

<details>
  <summary>Full File (Very Long)</summary>
const fs = require('fs');

const { prefix, token, clientID, clientSecret, PARENT, HORSPERs, MVPs, VIPs, ULTRAs, SUPERs } = require('./config.json');

const Discord = require('discord.js');
const client = new Discord.Client();

const cooldowns = new Discord.Collection();

client.commands = new Discord.Collection();

const commandFiles = fs.readdirSync('./commands').filter(file => file.endsWith('.js'));

for (const file of commandFiles) {
    const command = require(`./commands/${file}`);

    client.commands.set(command.name, command);
}

client.once('ready', () => {
    client.user.setActivity(`${prefix}help`, { type: 'PLAYING' });
    console.log('Ready!');
});

client.on('message', message => {
    if (message.webhookID || message.author.bot) return;

    if (message.content === 'horsey') {
        return message.react('742826710453452911');
    }

    if (message.content === 'HORSEY') {
        return message.react('800455575170711553');
    }

    const horseyBotMentionCheck = message.mentions.has(client.user) && !message.mentions.everyone;

    if (!message.content.startsWith(prefix) && !horseyBotMentionCheck) return;

    const args = message.content.slice(prefix.length).trim().split(' ');
    const commandName = args.shift().toLowerCase();

    const command = client.commands.get(commandName)
        || client.commands.find(cmd => cmd.aliases && cmd.aliases.includes(commandName));

    if (!command) {
        if (horseyBotMentionCheck) {
            let verified = 'Nah...';

            if (client.user.verified) verified = 'Yep!!!';

            const embed = new Discord.MessageEmbed()
                .setColor('RANDOM')
                .setAuthor(client.user.tag)
                .setThumbnail(client.user.displayAvatarURL({ format: 'png', dynamic: true }))
                .setTimestamp(Date.now())
                .setTitle('**Horsey!**')
                .setDescription(`My prefix is \`${prefix}\`. Have fun (specifically, run \`${prefix}help\` to get a list of commands as well as tips that'll elaborate the rest)!`)
                .addField('*Am I verified yet?*', verified, false)
                .setFooter(`Triggered by ${message.author.tag}`, message.author.displayAvatarURL({ format: 'png', dynamic: true }));
    
            return message.channel.send(embed);
        } else {
            return;
        }
    }

    if (command.guildOnly && message.channel.type === 'dm') {
        return message.reply('**Horsey!** That command is not supported in DMs.');
    }

    if (message.channel.type !== 'dm') {
        if (!message.guild.me.hasPermission('VIEW_CHANNEL') || !message.guild.me.hasPermission('SEND_MESSAGES')) {
            return console.log(`I can't send messages in ${message.guild.name}!`);
        }
    }

    if (command.args && !args.length) {
        let reply = `Horseyer ${message.author}, you need to provide some arguments!`;

        if (command.usage) {
            reply += `\nTo use this command like a horsey master, use it like this: \`${prefix}${command.name} ${command.usage}\``;
        }

        return message.channel.send(reply);
    }

    if (!cooldowns.has(command.name)) {
        cooldowns.set(command.name, new Discord.Collection());
    }

    const now = Date.now();
    const timestamps = cooldowns.get(command.name);
    const cooldownAmount = (command.cooldown || 1) * 1000;
    const cooldownAmountSUPER = (command.cooldown - 5 || 0.5) * 1000;
    const cooldownAmountULTRA = (command.cooldown - 10 || 0.5) * 1000;
    const cooldownAmountVIP = (command.cooldown - 15 || 0) * 1000;
    const cooldownAmountMVP = (command.cooldown - 25 || 0) * 1000;
    const cooldownAmountHORSPER = 0;

    let rank = 'HORSEYER';

    for (const SUPER of SUPERs) {
        if (message.author.id === SUPER) {
            rank = 'SUPER';
        }
    }

    for (const ULTRA of ULTRAs) {
        if (message.author.id === ULTRA) {
            rank = 'ULTRA';
        }
    }

    for (const VIP of VIPs) {
        if (message.author.id === VIP) {
            rank = 'VIP';
        }
    }

    for (const MVP of MVPs) {
        if (message.author.id === MVP) {
            rank = 'MVP';
        }
    }

    for (const HORSPER of HORSPERs) {
        if (message.author.id === HORSPER) {
            rank = 'HORSPER';
        }
    }

    if (message.author.id === PARENT) {
        rank = 'PARENT';
    }

    if (timestamps.has(message.author.id)) {
        let expirationTime = timestamps.get(message.author.id) + cooldownAmount;

        if (rank === 'HORSPER' || rank === 'PARENT') {
            expirationTime = timestamps.get(message.author.id) + cooldownAmountHORSPER;
        } else if (rank === 'MVP') {
            expirationTime = timestamps.get(message.author.id) + cooldownAmountMVP;
        } else if (rank === 'VIP') {
            expirationTime = timestamps.get(message.author.id) + cooldownAmountVIP;
        } else if (rank === 'ULTRA') {
            expirationTime = timestamps.get(message.author.id) + cooldownAmountULTRA;
        } else if (rank === 'SUPER') {
            expirationTime = timestamps.get(message.author.id) + cooldownAmountSUPER;
        }

        if (now < expirationTime) {
            const timeLeft = (expirationTime - now) / 1000;

            let response = `You must wait ${timeLeft.toFixed(2)} more second(s)!`;

            if (command.cooldownMessage) {
                response = `${command.cooldownMessage}, you must wait ${timeLeft.toFixed(2)} more second(s)!`;
            }

            return message.channel.send(response);
        }
    }

    try {
        timestamps.set(message.author.id, now);
        setTimeout(() => timestamps.delete(message.author.id), cooldownAmount);
    } catch (error) {
        console.error('500 - ill horsey\n', error);
        message.reply('HORSEY? 500 - ill horsey; I messed something up.');
    }

    try {
        command.execute(message, args);
    } catch (error) {
        console.error('400 - bad horsey\n', error);
        message.reply('HORSEY? 400 - bad horsey; you messed something up.');
    }
});

client.on('shardError', error => {
    console.error('A websocket connection encountered an error:', error);
});

process.on('unhandledRejection', error => {
    console.error('Unhandled promise rejection:', error);
});

client.on('debug', console.debug);

client.login(token);
</details>

**Comment by Androz2091** [comment, marked as answer]
Hello,
If you want to check if your bot is verified you should use the user flags property.
```js
let verified = 'Nah...';

if (client.user.flags.has('VERIFIED_BOT')) verified = 'Yep!!!';
```
